database/migrations/2023_10_04_085425_tasks.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('tasks', function (Blueprint $table) {
+            $table->id();
```

Comment:
Maybe, but bigint will have no problem to accompany the amount of tasks in a relatively small system like CC. I think we should rather look into uuid in a seperate PR and establish a guideline on when and how to use it first in the whole application rather than shotgunning it here. So I'm scoping this out to avoid 🐰 🕳️ in this PR.